examples/angularjs_require/js/controllers/todo.js

```diff
@@ -52,6 +52,8 @@ define(['app', 'services/todoStorage'], function (app) {
 
 			$scope.editTodo = function (todo) {
 				$scope.editedTodo = todo;
+				// Clone the original todo to restore it on demand.
+				$scope.originalTodo = angular.extend({}, todo);
```

Comment:
It's like this in other apps as well, but changing. https://github.com/tastejs/todomvc/blob/master/examples/angularjs/js/controllers/todoCtrl.js#L55